moped-editor/src/utils/activityLogFormatters/mopedProjectActivity.js

```diff
@@ -112,6 +138,38 @@ export const formatProjectActivity = (change, lookupList) => {
       };
     }
 
+    if (updatedFullName) {
+      return {
+        changeIcon,
+        changeText: [
+          {
+            text: "Changed full project name to ",
+            style: null,
+          },
+          {
+            text: changeData.new["project_name_full"],
+            style: "boldText",
+          },
+        ],
+      };
+    }
+
+    if (changedField === "project_name_full") {
```

Comment:
Everything works great, but why is this checking for the full name field when it is generating text about the project name?